public/js/publish/entity-handler.js

```
@@ -0,0 +1,146 @@
+    this.isLoadingGroups = false;
+    this.onChange();
+  }
+
+  async setAccountName(accountName) {
+    if (this.accountName !== accountName) {
+      this.accountName = accountName;
+      
+      this.activeAccount = this.accounts?.find(a => a.accountName === this.accountName);
+      await this.onAccountNameChanged();
+    }
+  }
```

Biome (2.1.2)

[error] Duplicate class member name "setAccountName"



(lint/suspicious/noDuplicateClassMembers)